### ISSUE TITLE
Problems in `morpheme-mappings.xml`

### ISSUE THREAD

**Issue by themikejr** [Clear-Bible/macula-hebrew#75]
The dashboard team is going to be relying on the [morpheme-mapping](https://github.com/Clear-Bible/macula-hebrew/blob/main/mappings/morpheme-mappings.xml) file so that ClearEngine can utilize the existing verse-based trees while the rest of the Dashboard can rely on the active work happening in the lowfat trees.

While looking at the aforementioned mapping file, I noticed the following issues:

* The attribute that represents the current lowfat identifiers seems to be called `n`. Conversely, I expected to find an `id` or `xml:id` attribute which represents the lowfat trees in `main`.
* The values that represent the current lowfat identifiers seem to be out of date. They do not have the leading `o` character and I did not see an instances where trailing hebrew consonants are included for handling the implied article.

There also may be other issues that were not obvious to me.

**Comment by jonathanrobie** [comment]
Yes, I think these are both bugs.  The attribute should be `xml:id` and it should start with `n` in the New Testament.

**Comment by jonathanrobie** [comment]
## Proposed fix

`mappings/morpheme-mappings.xml` uses `@n` as the identifier attribute, with values that lack the `o` prefix used in the nodes XML:

```xml
<!-- morpheme-mappings.xml -->
<m n="010010010011" marble="00100100100002" morphId="010010010011"/>

<!-- WLC/nodes/01-Gen-001.xml -->
<m xml:id="o010010010011" .../>
```

Two problems:
1. The attribute is named `n` instead of `xml:id` (or at minimum `id`) — makes it hard to JOIN against nodes data
2. Values lack the leading `o` — so `n="010010010011"` does not match `xml:id="o010010010011"`

### Fix

Rename `@n` → `@xml:id` and add the `o` prefix to all values:

```xml
<m xml:id="o010010010011" marble="00100100100002" morphId="010010010011"/>
```

This can be done with a simple XSLT or sed transform:
```xslt
<xsl:attribute name="xml:id" select="concat('o', @n)"/>
```

Also note that the `@morphId` column duplicates `@n` exactly — if `@n` is renamed to `@xml:id`, `@morphId` becomes redundant and could be removed.
